site/en/configure/json-trace-profile.md

```diff
@@ -0,0 +1,196 @@
+Project: /_project.yaml
+Book: /_book.yaml
+
+# JSON Trace Profile
+
+{% include "_buttons.html" %}
+
+The JSON trace profile can be very useful to quickly understand what Bazel spent
+time on during the invocation.
+
+By default, for all build-like commands and query Bazel writes such a profile to
+`command.profile.gz`. You can configure whether a profile is written with the
+[`--generate_json_trace_profile`](/reference/command-line-reference#flag--generate_json_trace_profile)
+flag, and the location it is written to with the
+[`--profile`](/user-manual#profile) flag. Locations ending with `.gz` are
```

Comment:
The previous docs code used `/user-manual` - which is more helpful - linking to the command line reference, or the user manual? For most cases, I used the command line reference (also because that has some entries the user manual does not), but I saw that at least in some cases the entry in the user manual gives helpful information not included in the command line reference. I'm fine with including either - the command line reference seems more consistent, but if the user manual provides more value, perhaps that's the way to go / keep it.